modules/renter/renter.go

```diff
@@ -338,6 +339,29 @@ func (r *Renter) ProcessConsensusChange(cc modules.ConsensusChange) {
 	r.mu.Unlock(id)
 }
 
+// validateSiapath checks that a Siapath is a legal filename.
+// ../ is disallowed to prevent directory traversal, and paths must not begin
+// with / or be empty.
+func validateSiapath(siapath string) error {
+	if strings.HasPrefix(siapath, "/") {
+		return errors.New("siapath cannot begin with /")
+	}
+
```

Comment:
no need for all of these newlines between each `if` block, can just condense the whole function.

Reply:
I'll take it out, I didn't write of modify this function at all.  I simply moved it from `modules/renter/upload.go` to `modules/renter/renter.go`.  If you don't like the formatting, blame yourself or Luke.  :)

Reply:
no blame, only fix :)